### ISSUE TITLE
'grin server start' does not progress

### ISSUE THREAD

**Issue by kargakis** [mimblewimble/grin#947]
grin hangs forever with the following log
```
Apr 08 21:34:05.287 INFO This is Grin version 0.2.0 (git testnet2-48-g65d5bd4), built for armv7-unknown-linux-gnueabihf by rustc 1.25.0 (84203cac6 2018-03-25).
Apr 08 21:34:05.288 DEBG Built with profile "release", features "" on Sat, 07 Apr 2018 00:58:50 GMT.
Apr 08 21:34:05.288 INFO Starting the Grin server from configuration file at /home/pi/grin-image/grin.toml
Apr 08 21:34:05.288 DEBG Using wallet seed file at: ./wallet.seed
Apr 08 21:34:05.626 INFO Starting the Grin wallet receiving daemon at 127.0.0.1:13415...
Apr 08 21:34:05.632 INFO Wallet listener started
```
`grin server run` works as expected.

**Comment by kargakis** [comment]
Still seeing this

**Comment by ignopeverell** [comment]
Should have been fixed by ce1dca1. I would also recommend syncing on a release build to avoid too long delays validating the state.
